classes/Policies.php

```diff
@@ -303,4 +313,37 @@ public function limitToSet($set) {
         }
     }
 
+    public function getPolicyDetails($policyID) {
+        $q = $this->db->query(
+            "SELECT policy_id, title, description, image, image_attrib, image_license, image_license_url, image_source
+            FROM policies WHERE policy_id = :policy_id",
+            array(':policy_id' => $policyID)
+        );
+
+        $props = array(
+            'policy_id' => $q->field(0, 'policy_id'),
+            'title' => $q->field(0, 'title'),
+            // remove full stops from the end of descriptions. Some of them have them and
+            // some of them don't so we enforce consistency here
+            'description' => preg_replace('/\. *$/', '', $q->field(0, 'description')),
+            'image' => '/images/header-debates-uk.jpg', // TODO: get a better default image
+            'image_license' => $q->field(0, 'image_license'),
+            'image_attribution' => $q->field(0, 'image_attrib'),
+            'image_source' => $q->field(0, 'image_source'),
+            'image_license_url' => $q->field(0, 'image_license_url')
+        );
+
+        $image = $q->field(0, 'image');
+
+        if ( $image && file_exists(BASEDIR . '/' . $image)) {
+            $props['image'] = $image;
+            $props['image_license'] = $q->field(0, 'image_license');
```

Comment:
These four lines repeat what is done above; not sure if these should be removed, or if the four above should be changed to default data?